### PR TITLE
#159316712 Hosting on Heroku

### PR DESCRIPTION
**Pull request by tutugodfrey**
#### What does this PR do?
- Add configurations to host app on heroku

#### Description of Task to be completed?
- create heroku pipeline
- create heroku apps for staging and production
- update default app.json
- add Procfile and script for starting the application

#### How should this be manually tested?
- heroku pipeline will generate a review app on everypull request
- the current state of the application can be tested from the generated review app

#### What are the relevant pivotal tracker stories?
#159316712

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 const router = require("express").Router();
 
 router.use("/api", require("./api"));
+// return for call to /api
+router.get("/api", (req, res) => {
```
Comment (megame24) [inline routes/index.js]: Why did you edit this route?

File: Procfile [hunk 2]
```diff
@@ -0,0 +1 @@
+web: node index.js
```
Comment (chidioguejiofor) [inline Procfile]: Add New Line at the end of file

**Review by chidioguejiofor — CHANGES_REQUESTED** [review]
Please modify your PR to add the link to the Heroku app and also add some screenshots that show you consuming some of the endpoints from the Heroku app link. 
Once this is done notify me to approve your PR

File: routes/index.js (hunk 1 above)
Comment (megame24) [inline routes/index.js]: Why did you edit this route? This update will cut off entry to all the other routes and controllers, please attend to it.

**Review by megame24 — COMMENTED** [review]
Please attend to chidi's request.

File: routes/index.js (hunk 1 above)
Comment (megame24) [inline routes/index.js]: Why did you edit this route? By updating this route, you have cut off the rest of the routes and controllers. Please attend to it by reversing the changes.

**Review by megame24 — APPROVED** [review]

**Pull request closed without merging**